cli/src/pcluster/cli/commands/cluster.py

```diff
@@ -215,7 +213,7 @@ class ListClustersCommandV2(CliCommand):  # TODO: to be removed
     name = "list"
     help = "Displays a list of stacks associated with AWS ParallelCluster."
     description = help
-    epilog = f"This command lists the names of any CloudFormation stacks named {PCLUSTER_STACK_PREFIX}*"
+    epilog = f"This command lists the names of any CloudFormation stacks with tag {PCLUSTER_VERSION_TAG}"
```

Comment:
help = "Displays a list of clusters created by AWS ParallelCluster."
epilog, ok.

Reply:
Done